.github/workflows/ci.yml

```diff
@@ -20,12 +20,12 @@ jobs:
     strategy:
       matrix:
         # Test of these containers
-        container: ["ubuntu-dev:20", "alpine-dev:latest"]
-        compiler: [{cxx: g++, c: gcc}]
-        cxx_flags: ["-Werror"]
         include:
           - container: "alpine-dev:latest"
             compiler: {cxx: clang++, c: clang}
+          - container: "ubuntu-dev:20"
```

Comment:
Coverage collection tends to have a large perf impact and is sometimes unstable - I had a few random crashes from (presumably) compiler bugs when I ran coverage over the DFLY test suite. So I'm a bit uncomfortable with doing this as a 'normal' CI run.. But then also if it works it works

Reply:
That's why I did it on helio. Lets see how it goes.